src/RA_Core.cpp

```diff
@@ -1636,7 +1626,7 @@ std::string GetFolderFromDialog()
     std::string sRetVal;
 	CComPtr<IFileOpenDialog> pDlg;
 
-    HRESULT hr;
+    HRESULT hr = HRESULT{};
```

Comment:
Didn't want to guess what "0" was for HRESULT. It's like a lot of other languages but without the new.